wiring/src/spark_wiring_usbserial.cpp

```diff
@@ -24,6 +24,11 @@
   ******************************************************************************
  */
 
+#ifdef __clang__
```

Comment:
This does not need to be `__clang__`-specific.

Reply:
Fixed